kotlin-native/shared/src/main/kotlin/org/jetbrains/kotlin/konan/util/DependencyProcessor.kt

```diff
@@ -96,10 +96,17 @@ class DependencyProcessor(dependenciesRoot: File,
                           private val deleteArchives: Boolean = true,
                           private val archiveType: ArchiveType = ArchiveType.systemDefault) {
 
-    private val dependenciesDirectory = dependenciesRoot.apply { mkdirs() }
-    private val cacheDirectory = homeDependencyCache.apply { mkdirs() }
+    private val dependenciesDirectory by lazy {
+        dependenciesRoot.apply { mkdirs() }
+    }
 
-    private val lockFile = File(cacheDirectory, ".lock").apply { if (!exists()) createNewFile() }
+    private val cacheDirectory by lazy {
+        homeDependencyCache.apply { mkdirs() }
+    }
+
+    private val lockFile by lazy {
+      File(cacheDirectory, ".lock").apply { if (!exists()) createNewFile() }
```

Comment:
Wrong indent.

Reply:
Thanks, updated in 8082b5895fd8c3cc95767be68e4e397a931ca338